docs/compatibility.md

```diff
@@ -0,0 +1,15 @@
+# Compatibility with Previous Versions of MMDetection3D
+
+This document provides detailed descriptions of the BC-breaking changes in MMDetection3D.
+
+## MMDetection3D 0.12.0
+
+### SUNRGBD dataset for ImVoteNet
```

Comment:
I remember there is a BC-breaking regarding VoteNet model and its pre-trained weight. Should we also mention it here?

Reply:
Can you point me to that pull request?

Reply:
I am not sure about the PR link, but if you look at the VoteNet's page [here](https://github.com/open-mmlab/mmdetection3d/blob/master/configs/votenet/README.md) (**Result** section, **Notice** part), you can see instruction about converting pre-trained model weight.

Reply:
This #163 seems related, @ZwwWayne what do you think?

Reply:
Yes, but isn't this too ancient?

Reply:
Not sure if that's too ancient. At least it seems indeed a BC-breaking to me. And when I first use MMDet3D, I forgot to convert the pre-trained weight lol. It's okay if you don't add it.

Reply:
I agree. I'll add it :)